src/Documents.ts

```diff
@@ -776,6 +776,9 @@ export class Class implements Pointer.Library, Pointer.Validator, ObjectSchema.R
 				persistedProtectedDocument,
 				response,
 			];
+		}, ( error ) => {
```

Comment:
To improve readability please use a catch statement instead of sending the error function as a second parameter.